pulsar-common/src/main/java/org/apache/pulsar/common/naming/TopicName.java

```diff
@@ -288,6 +288,13 @@ public boolean isGlobal() {
         return cluster == null || Constants.GLOBAL_CLUSTER.equalsIgnoreCase(cluster);
     }
 
+    public String getSchemaName() {
+        return getProperty()
+            + "_" + getCluster()
```

Comment:
Since `_` is a valid character for property namespace and such, it will be difficult to decode the schema name into a topic name. I would say to stick with the current convention of `/` as the divider, unless there is a specific reason.

Reply:
zk tends to be unhappy with the '/' since it thinks I'm referencing a path, any other suggestions.

Reply:
ok, will recursively create directories as needed.